variables.tf

```diff
@@ -205,3 +205,13 @@ variable "boot_disk_kms_key" {
   description = "CloudKMS key_name to use to encrypt the nodes boot disk. Default: null (encryption disabled)"
   default     = null
 }
+
+variable "kubelet_config" {
```

Comment:
Can you add a comment with a link to the documentation? So that we know what are the supported values (and meanings)

Reply:
Added at https://github.com/Devgurusio/terraform-gcp-gke-ecommerce/pull/9/commits/b8cff4c2e2719a7ec8b0e4934e56ef79b3202420